src/components/capabilities/implementation/fission-lobby.ts

```diff
@@ -99,26 +85,26 @@ export async function request(
 
   // Compile params
   const params = [
-    [ "didExchange", exchangeDid ],
-    [ "didWrite", writeDid ],
-    [ "redirectTo", redirectTo ],
-    [ "sdk", VERSION.toString() ],
-    [ "sharedRepo", sharedRepo ? "t" : "f" ],
-    [ "sharing", sharing ? "t" : "f" ]
+    ["didExchange", exchangeDid],
+    ["didWrite", writeDid],
+    ["redirectTo", redirectTo],
+    ["sdk", VERSION.toString()],
+    ["sharedRepo", sharedRepo ? "t" : "f"],
+    ["sharing", sharing ? "t" : "f"]
 
   ].concat(
-    app ? [ [ "appFolder", `${app.creator}/${app.name}` ] ] : [],
-    fs?.private ? fs.private.map(p => [ "privatePath", Path.toPosix(p, { absolute: true }) ]) : [],
-    fs?.public ? fs.public.map(p => [ "publicPath", Path.toPosix(p, { absolute: true }) ]) : [],
-    raw ? [ [ "raw", Base64.urlEncode(JSON.stringify(raw)) ] ] : [],
```

Comment:
Really need to add a commit hook with some default formatting 😅 I switched editors.